### PR TITLE
enhance: new messsage interface for log service

### PR DESCRIPTION
**Pull request by chyezh**
issue: #33285

### CODE REVIEW

**Comment by chyezh** [comment]
rerun ut

File: internal/util/logserviceutil/message/message_type.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package message
+
+import "strconv"
+
+type MessageType int32
+
+const (
+	MessageTypeUnknown  MessageType = 0
+	MessageTypeTimeTick MessageType = 1
```
Comment (bigsheeper) [inline internal/util/logserviceutil/message/message_type.go:9]: Why do we need only the time tick message type?

**Comment by bigsheeper** [comment]
Just don't forget to remove all invocations to panic in the future.

**Comment by chyezh** [comment]
rerun ut

**Pull request merged**